presto-parser/src/main/java/com/facebook/presto/sql/tree/ColumnDefinition.java

```diff
@@ -29,24 +29,25 @@
     private final String type;
     private final List<Property> properties;
     private final Optional<String> comment;
+    private final boolean nullable;
```

Comment:
Maybe we could provide an enum for that, that way code would be much more readable.

Reply:
Not sure I agree this will help a lot with readability.  Also do we propagate this to SPI etc?  It seems this remaining as a boolean will be consistent and less surprising--there's only ever two states to whether or not a column is nullable.

Reply:
If the problem with readability is the boolean flag that's being set in the constructor, perhaps later we can also make this class a builder (like we will with `ColumnMetadata`).